Package.swift

```diff
@@ -23,6 +23,14 @@ let swiftSystem: PackageDescription.Target.Dependency = .product(
   condition: .when(platforms: [.macOS, .iOS, .tvOS, .watchOS, .linux, .android])
 )
 
+// This doesn't work when cross-compiling: the privacy manifest will be included in the Bundle and
+// Foundation will be linked. This is, however, strictly better than unconditionally adding the
+// resource.
+#if canImport(Darwin)
```

Comment:
This is evaluated on the wrong platform when cross compiling. 

CC @MaxDesiatov , is there a `.when(...)` platform thing?

Reply:
Correct, and as noted in the comment, if we don't do this the alternative is unconditionally including the resource which is worse.

There's no `.when` condition for resources but other solutions gladly accepted!

Reply:
Yes, this will break when cross-compiling, but apparently `.when` is not flexible enough either 😞